backend/request

```diff
@@ -0,0 +1,17 @@
+POST http://localhost:3000/sensorSubmit HTTP/1.1
+content-type: application/json
+
+{
+    "id": 12348,
+    "location": "B5",
+    "temperature": 19.5,
+    "time": "01:02:03T20:12:2020"
+}
+
+# POST http://localhost:3000/vote HTTP/1.1
+# content-type: application/json
+
+# {
+#     "submitter": "John",
+#     "opinion": -1
+# }
```

Comment:
What is this for?

Reply:
That's for the REST extension for VS Code. If you press CTRL + ALT + R in that file, it'll send a HTTP request with the specified data